docs/firmware/device-sdk/light-db-stream/concepts-lightdb-stream.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Concepts - LightDB Stream

```

Comment:
These are fine to leave for now but would like them to drop the "Concepts - " and be more descriptive, like "Sending logs."